cmd/internal/kube/print_fragment.go

```diff
@@ -0,0 +1,149 @@
+package kube
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/postmanlabs/postman-insights-agent/cfg"
+	"github.com/postmanlabs/postman-insights-agent/cmd/internal/cmderr"
+	"github.com/postmanlabs/postman-insights-agent/rest"
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/spf13/cobra"
+	"github.com/zclconf/go-cty/cty"
+	"sigs.k8s.io/yaml"
+
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+)
+
+var printHelmChartFragmentCmd = &cobra.Command{
+	Use:              "helm-fragment",
+	Short:            "Print a Helm chart container definition for adding the Postman Insights Agent to existing k8s deployment.",
+	Long:             "Print a container definition that can be inserted into a Helm Chart template to add the Postman Insights Agent as a sidecar container.",
+	RunE:             printHelmChartFragment,
+	PersistentPreRun: kubeCommandPreRun,
+}
+
+var printTerraformFragmentCmd = &cobra.Command{
+	Use:              "tf-fragment",
+	Short:            "Print an Terraform (HCL) code fragment for adding the Postman Insights Agent to an existing k8s deployment.",
```

Comment:
I think we should avoid the numeronym k8s in our output, even if it's OK to accept it as an alias.